iron-spi-file-inttest/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
to be merged into iron-core

Reply:
But that creates a module cycle:
iron-core (test) depends on iron-sample
iron-sample depends on iron-core

Reply:
yes, as discussion we will create a single iron-inttest module

Reply:
Fixed.